src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_img2img.py

```diff
@@ -820,6 +839,21 @@ def __call__(
 
         # 9. Denoising loop
         num_warmup_steps = len(timesteps) - num_inference_steps * self.scheduler.order
+
+        # 9.1 Apply denoising_end
+        if denoising_end is not None and denoising_start is not None:
+            if denoising_end <= denoising_start:
```

Comment:
End values are usually larger than the start values, but that is not the case here. Let's make sure we are aligned on what `denoising_start` means.

Reply:
I don't understand this comment. If denoising start is larger equal end, we throw an error

Reply:
I am confused by what `denoising_start` means here. 

Usually, the end values of something tend to be larger (in terms of quantity) than the initial (start) values. 

So, maybe clarifying what `denoising_start` means will help me understand this better.

Reply:
I now changed the order, now so that the error is easier to understand, but I don't follow what is the problem here. 

If you want to finish denoising at 40% but you want to start at 50% it doesn't work obviously so it should then error out.